esp/GBPlay/main/hardware/wifi.c

```diff
@@ -1,3 +1,4 @@
+#include <assert.h>
```

Comment:
I suggest turning on "hide whitespace" for `wifi.c`. I changed the indentation to better denote the sections where locks are held.